Week3/homework/src/actions/removeTodoList.js

```diff
@@ -0,0 +1,17 @@
+'use-strict';
+
+const connection = require('../databaseConfig');
+
+const removeTodoList = (req, res) => {
+  connection.query(`DELETE FROM todolists WHERE ToDoList_Id = '${req.params.todoListId}'`);
```

Comment:
Non-blocking comment:

There is a good practice to use LIMIT in your queries: https://dba.stackexchange.com/questions/52079/does-using-limit-improve-the-performance-and-is-it-noticeable

Long story short:

- in case of select of only one row, you want to return the result once the row is found because you're looking for only one row, and it would be wise to stop the search when the one is found. However, if you select by PK - SQL server will know it and will stop the search itself - because of PK is unique
- in case of deletion - it's good for security reasons, and also saves you from occasional deleting of multiple rows.

Reply:
Done :pray: